lib/booktype/skeleton/base_settings.py.original

```diff
@@ -34,20 +34,21 @@ BOOKTYPE_SITE_NAME = None
 BOOKTYPE_SITE_DIR = '##BOOKTYPE_SITE_DIR##'
 
 # The hostname of the Booktype server (e.g. www.myserver.org, 192.168.1.10, booktype.myserver.org)
-THIS_BOOKTYPE_SERVER = ''
+THIS_BOOKTYPE_SERVER = os.environ['BOOKTYPE_SERVER'] if 'BOOKTYPE_SERVER' in os.environ else ''
```

Comment:
Hi @ride90,
most of the lines in the PR looks a bit redundant and they could be improved (and will look more elegant) using something like this:
`THIS_BOOKTYPE_SERVER = os.environ.get("BOOKTYPE_SERVER", "DEFAULT_VALUE")`
Please update the PR to merge it after :)